src/main/groovy/com/netflix/nebula/lint/plugin/LintService.groovy

```diff
@@ -96,14 +92,14 @@ class LintService {
             if (!excludedRules.isEmpty())
                 includedRules.retainAll { !excludedRules.contains(it.name) }
 
-            return RuleSetFactory.configureRuleSet(includedRules)
+            return RuleSetFactory.configureRuleSet(includedRules + createRuleForApplyFiles(p, analyzer))
         }
         return new ListRuleSet([])
     }
 
     RuleSet ruleSet(Project project) {
         def ruleSet = new CompositeRuleSet()
-        ([project] + project.subprojects).each { p -> ruleSet.addRuleSet(ruleSetForProject(p)) }
+        ([project] + project.subprojects).each { p -> ruleSet.addRuleSet(ruleSetForProject(p, NO_ANALYZER)) }
```

Comment:
Here I'm not completely sure. I found that rules are shown in a generated report so they are not evaluated. I will probably need to provide set of rules without internal recursive rule.